athena/tests/module_text_llm/test_basic_approach_mock.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from module_text_llm.basic_approach.generate_suggestions import generate_suggestions
+from athena.text import Exercise, Submission, Feedback
+from athena.schemas.exercise_type import ExerciseType
+from tests.utils.mock_env import mock_sent_tokenize
+from tests.utils.mock_llm import MockLanguageModel, MockAssessmentModel, MockFeedbackModel
+
+@pytest.fixture
+def mock_exercise():
+    """Create a mock exercise for testing."""
+    return Exercise(
+        id=1,
+        title="Test Exercise",
+        type=ExerciseType.text,
+        max_points=10,
+        bonus_points=2,
+        grading_instructions="Test grading instructions",
+        problem_statement="Test problem statement",
+        example_solution="Test example solution",
+        grading_criteria=[]
+    )
+
```

Comment:
Can you please adapt the PEP8 style to all files? Otherwise, viewing those files in my IDE is quite colorful:
<img width="734" alt="image" src="https://github.com/user-attachments/assets/7090b99c-f8f5-4658-81a9-f1537fb6feb1" />